cclib/parser/nwchemparser.py

```diff
@@ -605,6 +630,12 @@ def extract(self, inputfile, line):
         if "Dispersion correction" in line:
             dispersion = utils.convertor(float(line.split()[-1]), "hartree", "eV")
             self.append_attribute("dispersionenergies", dispersion)
+            
+        # type of dispersion
+        if line.strip().find('DFT-D3 Model') > -1:
+            self.metadata['dispersion'] = "D3"

```

Comment:
Can we do `D3(0)` and `D3(BJ)`?  (unless there is a better way to resolve the ambiguity)

Reply:
yes, absolutely. good call.